server/app/service/ScalaTestRunner.scala

```diff
@@ -18,6 +24,22 @@ object ScalaTestRunner {
   val traitDefPattern = """trait\s*([\w\$]*)""".r
   val defaultImports = "import org.scalatest._"
 
+  def apply(suiteClass: Class[Suite], solutionTrait: Class[AnyRef])(solution: String)(implicit s: Scheduler): Observable[Line] = {
```

Comment:
Copy that, at the moment it is just the contract that you asked, it works wrong, looks wrong and located at wrong place too